OneMore/OneNote.cs

```diff
@@ -1303,18 +1320,57 @@ public async Task NavigateTo(string pageId, string objectId)
 			});
 		}
 
-
-		/// <summary>
-		/// Search pages under the specified hierarchy node using the given query.
-		/// </summary>
-		/// <param name="nodeId">
-		/// Can be String.Empty for all notebooks, CurrentNotebookId, CurrentSectionId,
-		/// or CurrentPageId
-		/// </param>
-		/// <param name="query">The search string</param>
-		/// <param name="unindexed">True to include unindexed pages in query</param>
-		/// <returns>An hierarchy of pages whose content matches the search string</returns>
-		public XElement Search(string nodeId, string query, bool unindexed = false)
+        /// <summary>
+        /// Forces OneNote to move a page after one another existed page.
+        /// </summary>
+        /// <param name="CurPageId">Current page ID</param>
+        /// <param name="DestPageId">Destination page ID</param>
+        /// <returns></returns>
+        public async Task MovedAfterPage(string CurPageId, string DestPageId)
+        {
+            var root = GetAllPages();
```

Comment:
This is very inefficient. There are easier ways of discovering, tracking, and using the notebook and section IDs along with the page and target IDs to make this faster.